### PR TITLE
codex/improve-dual-channel-command-ux

### PR DESCRIPTION
**Pull request by Dj-Shortcut**
## Summary
- localize the assistant command copy used across Messenger and WhatsApp
- improve WhatsApp help text so users without quick replies get clear plain-text guidance
- use user-facing style labels for the random style command instead of internal slugs

## Verification
- `pnpm vitest run server/whatsappWebhook.test.ts server/messengerWebhook.test.ts`
- `pnpm check`

## Summary by CodeRabbit

* **New Features**
  * Assistant messages fully localized (EN/NL): quick-actions prompt, multi-paragraph photo guidance, and “surprise me” style confirmation now use translated text and show a user-facing style label.

* **Tests**
  * Added/updated tests to validate localized assistant flows, photo + “surprise me” interactions, deterministic style selection messaging, and language-specific responses across channels.

### CODE REVIEW

File: server/_core/bot/features/assistantCommandsFeature.ts [hunk 1]
```diff
@@ -27,18 +28,28 @@ function pickRandomStyle() {
...
         );
       } else {
         await ctx.sendText(
-          `${t(ctx.lang, "textWithoutPhoto")}\n\nTip: send 'surprise me' after uploading a photo for an instant random style.`
+          [
+            t(ctx.lang, "textWithoutPhoto"),
+            t(ctx.lang, "assistantPhotoTip"),
+            ...(ctx.capabilities.quickReplies
+              ? []
+              : [t(ctx.lang, "assistantPhotoTipExtra")]),
```
[the remark this answers is not in the document]
Comment (Dj-Shortcut) [reply server/_core/bot/features/assistantCommandsFeature.ts]: Fixed in `3172a67`.

I kept the unit test update, but also added the defensive product-code guard here so `assistantCommandsFeature` no longer assumes `ctx.capabilities` is always present. That covers both the failing test context and any similar edge cases.

[the remark this answers is not in the document]
Comment (Dj-Shortcut) [reply server/_core/bot/features/assistantCommandsFeature.ts]: Fixed in `a8f49a1`.

I removed that extra pre-photo WhatsApp hint so the help text no longer promises style-name or numbered-option selection before an image has actually been uploaded. The flow still points users to upload a photo first, which is what the current implementation supports.
